src/pages/AddPost.js

```diff
@@ -101,29 +138,63 @@ const AddPost = ({ navigation }) => {
                 });
                 setErrors({ ...errors, description: "" });
               }}
+              placeholder="Description"
             />
             <RichToolbar editor={richText} />
             <ErrorComponent errorMessage={errors.description} />
           </View>
-          <View className="mb-2">
-            <TouchableOpacity
-              className="py-3 px-4 bg-blue-500 rounded-md"
-              onPress={handleImageSelect}
-            >
-              <Text className="text-center text-white">Select Image</Text>
-            </TouchableOpacity>
+          <View className="mb-2 h-[120px] flex justify-center items-center bg-white border-gray-200 border rounded-lg">
+            {newPostData.picture ? (
```

Comment:
you should create a component for it, which will handle the image show and option to select image from folder or camera @amitark7